src/configurations/destinations/braze/schema.json

```diff
@@ -1055,6 +1062,26 @@
             }
           }
         }
+      },
+      "appKey": {
+        "type": "string",
+        "pattern": "^(.{1,100})$"
+      },
+      "androidApiKey": {
+        "type": "string",
+        "pattern": "^(.{1,100})$"
+      },
+      "iOSApiKey": {
+        "type": "string",
+        "pattern": "^(.{1,100})$"
+      },
+      "webApiKey": {
+        "type": "string",
+        "pattern": "^(.{1,100})$"
+      },
+      "usePlatformSpecificApiKeys": {
```

Comment:
The schema has to be updated to constraint that at least one of the api keys should be present when this flag is enabled.

Reply:
Schema generator is not perfect unfortunately.